javadoc-remote/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-parent</artifactId>
+      <version>9.0.0-SNAPSHOT</version>
+      <relativePath>../parent/pom.xml</relativePath>
+   </parent>
+   <artifactId>infinispan-javadoc-remote</artifactId>
+   <name>Infinispan JavaDoc Remote</name>
+   <packaging>jar</packaging>
+
+   <properties>
+      <analytics.code>
+---&gt;
+-&lt;!--
+-&lt;script type='text/javascript'&gt;
+-var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
+-document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
+-&lt;/script&gt;
+-&lt;script type='text/javascript'&gt;
+-try {
+-var pageTracker = _gat._getTracker("UA-8601422-4");
+-pageTracker._trackPageview();
+-} catch(err) {}&lt;/script&gt;
+            </analytics.code>
```

Comment:
We need to update the analytics code to the new async stuff:
`<script>
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
                        (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
                        m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
                        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
     
    ga('create', 'UA-8601422-1', 'auto');
    ga('send', 'pageview');
    </script>`